packages/cli/test/utils/simulation/execution_clients/geth.ts

```diff
@@ -166,11 +165,8 @@ export const generateGethNode: ExecutionNodeGenerator<ExecutionClient.Geth> = (o
 
   const job = runner.create([{...initJobOptions, children: [{...importJobOptions, children: [startJobOptions]}]}]);
 
-  const provider = new Eth1ProviderWithAdmin(
-    {DEPOSIT_CONTRACT_ADDRESS: ZERO_HASH},
-    // To allow admin_* RPC methods had to add "ethRpcUrl"
-    {providerUrls: [ethRpcPublicUrl, engineRpcPublicUrl], jwtSecretHex: SHARED_JWT_SECRET}
-  );
+  const web3 = new Web3(ethRpcPublicUrl);
```

Comment:
I think `provider` feels more industry standard in this context.  I tend to like the old way better so its clear what the object does to observers.

```suggestion
  const provider = new Web3(ethRpcPublicUrl);
```